dll/win32/shlwapi/ordinal.c

```diff
@@ -3288,6 +3291,32 @@ BOOL WINAPI PlaySoundWrapW(LPCWSTR pszSound, HMODULE hmod, DWORD fdwSound)
 DWORD WINAPI SHGetIniStringW(LPCWSTR appName, LPCWSTR keyName, LPWSTR out,
         DWORD outLen, LPCWSTR filename)
 {
+#ifdef __REACTOS__
+    WCHAR szSection[MAX_PATH + 2];
+    WCHAR szBuffW[512];
+    CHAR szBuffA[512];
+
+    TRACE("(%s,%s,%p,%08x,%s)\n", debugstr_w(appName), debugstr_w(keyName),
+          out, outLen, debugstr_w(filename));
+
+    if (outLen == 0)
+        return 0;
+
+    lstrcpynW(szSection, appName, _countof(szSection) - 2);
+    lstrcatW(szSection, L".W");
+
+    GetPrivateProfileStringW(szSection, keyName, L"\xFFFF", szBuffW, _countof(szBuffW), filename);
```

Comment:
Why? (missing explanation in comment)

Reply:
Added comments. Retired of using `0xFFFF`.